tests/schema_derivation_tests/query_spec_tests/operators_precedence.yml

```diff
@@ -883,7 +883,7 @@ tests:
   }
 - description: unary ops bind tighter than mul/div
   current_db: db
-  skip_reason: SQL-2752 improve $literal ergonomics
+  skip_reason: "negative integer literals deserialize as long"
```

Comment:
Here too, can you have a JIRA reference please.
This is just to make it easier to know that we are aware of it and if we ever fix this, it is easy to find the test and "unskip" it.